pkgs/development/python-modules/xmlschema/default.nix

```diff
@@ -1,22 +1,23 @@
 { lib, buildPythonPackage, fetchFromGitHub
 , elementpath
+, lxml
 , pytest
 }:
 
 buildPythonPackage rec {
-  version = "1.1.0";
+  version = "1.2.2";
   pname = "xmlschema";
 
```

Comment:
```suggestion
  disabled = pythonOlder "3.5";

```

Reply:
we will never have a python3 < 3.5 in nixpkgs. I think this is okay for now. But your suggestion is technically more correct